code/game/objects/items.dm

```diff
@@ -255,6 +255,11 @@
 	/// Do we apply a click cooldown when resisting this object if it is restraining them?
 	var/resist_cooldown = CLICK_CD_BREAKOUT
 
+	//BUBBER EDIT START
+	///How much this item is worth when part of a Hoard
+	var/hoard_value = 10
```

Comment:
If you don't need every item to have a hoard value you can use a component + 10 for all items seems iffy when that includes stuff like paper or other mass-volume items